src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -55,4 +74,15 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName, @Reque
         }
     }
 
+    private Optional<ValidationError> validateSchema(@RequestBody String event, EventType eventType) {
+        try {
+            ValidationStrategy validationStrategy = new EventBodyMustRespectSchema();
+            ValidationStrategyConfiguration vsc = new ValidationStrategyConfiguration();
```

Comment:
for me it seems strange that we create all that every time. Is it possible to create that only one time, and every time call only validationStrategy.materialize(eventType, vsc) ?